### PR TITLE
CRUMBS

### PR DESCRIPTION
**Pull request by henrylay97**
The functionality in sbndcode to run the new CRUMBS tool which lives in sbncode.

- CRUMBS is added as a producer to the standard and standard+sce caf workflows
- SCE versions of the crt hit and track matching are added as they are needed as inputs
- sbndcode is pointed to a new version of sbnd_data (v1_12_00) containing the weights file for CRUMBS's BDT

Note this PR assumed that #255 and #256 had already been merged into `release/SBN2021C`. Once this occurs then these extra commits should be removed from the diff.

Links to main PR SBNSoftware/sbncode#250

### CODE REVIEW

**Review by etyley — APPROVED** [review]
I can't really comment on the CRT changes, but the fcl/xml/ups changes all look reasonable to me

**Comment by henrylay97** [comment]
Perfect, thanks Ed! Yeah those CRT changes are hoovered up from #256 (which has been approved) and shouldn't be in this diff that's my bad, they will disappear as soon as that merged!

File: sbndcode/JobConfigurations/standard/caf/cafmakerjob_sbnd.fcl [hunk 1]
```diff
@@ -17,6 +17,7 @@
 #include "showerselectionvarsproducer.fcl"
 #include "showercosmicdistanceproducer.fcl"
 #include "sbn_pid.fcl"
+#include "crumbs_producer.fcl"
```
Comment (absolution1) [inline sbndcode/JobConfigurations/standard/caf/cafmakerjob_sbnd.fcl:20]: If this fcl lives in an SBN repo then may want to include 'sbn' in the name to indicate that it isn't in central larsoft

**Review by absolution1 — APPROVED** [review]
Looks good to me.
I left want comment about labelling but it can be addressed later.

**Pull request merged**